src/components/chartArea/chartArea.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import {
   Chart,
   ChartAxis,
+  ChartLine,
```

Comment:
should possibly become `ChartThreshold` in PF in the future... leaving this as a `ChartLine` for now...